dbt/include/athena/macros/materializations/models/configs.sql

```diff
@@ -0,0 +1,7 @@
+{% macro should_batch_inserts() %}
+  {% set config_batch_inserts = config.get('batch_inserts') %}
```

Comment:
```suggestion
  {% set config_batch_inserts = config.get('batch_inserts', False) %}
```
can we just replace this macro with such one-liner ?

Reply:
also I would prefer naming `force_batch_insert` instead for this parameter

Reply:
see this comment: https://github.com/dbt-athena/dbt-athena/pull/472#issuecomment-1776991626 yeah.
I like the name `force_batch_insert ` better, great suggestion.